core/templates/pages/admin-page/services/admin-router.service.ts

```diff
@@ -25,10 +25,18 @@ import { AdminPageConstants } from
 @Injectable({
   providedIn: 'root'
 })
+
```

Comment:
There shouldn't be any newline here, right? (In between decorator and class definition)

Reply:
Removed the space